causalml/inference/meta/rlearner.py

```diff
@@ -271,6 +304,63 @@ def bootstrap(self, X, p, treatment, y, size=10000):
         te_b = self.predict(X=X)
         return te_b
 
+    def run_propensity_score(self, X, treatment, X_pred=None, treatment_pred=None, cv=None,
+                             calibrate_propensity=True, return_model=False):
```

Comment:
`calibrate_propensity` --> `calibrate`. It's obvious that we're calibrating propensity scores.

Reply:
I changed to `calibrate_p` to be shorter because we import the same function here  `from causalml.propensity import calibrate`